crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -457,13 +457,13 @@ def order_instances(bcs)
 
   def self.read_options
     # read in default proposal, to make some vaules avilable
-    proposals = Proposal.where(barclamp: "crowbar")
-    raise "Can't find any crowbar proposal" if proposals.nil? or proposals[0].nil?
+    proposal = Proposal.find_by(barclamp: "crowbar")
+    raise "Can't find any crowbar proposal" if proposal.nil?
     # populate options from attributes/crowbar/*-settings
     options = { raid: {}, bios: {}, show: [] }
-    unless proposals[0]["attributes"].nil? or proposals[0]["attributes"]["crowbar"].nil?
-      options[:raid] = proposals[0]["attributes"]["crowbar"]["raid-settings"]
-      options[:bios] = proposals[0]["attributes"]["crowbar"]["bios-settings"]
+    unless proposal["attributes"].nil? or proposal["attributes"]["crowbar"].nil?
```

Comment:
Style/AndOr: Use `||` instead of `or`. (https://github.com/bbatsov/ruby-style-guide#no-and-or-or)